### PR TITLE
Bump Atlas on Tritium 0.8.4 to fix dependency conflicts

### PR DESCRIPTION
**Pull request by datval**
**Goals (and why)**:
Tritium was causing dependency conflict and some Atlas customers doesn't allow automatic conflict resolution.

**Implementation Description (bullets)**:
Bumped Tritium dependencies to be consistent with Atlas ones. Hopefully no more conflicts.

**Concerns (what feedback would you like?)**:
Did I miss something to bump?

**Where should we start reviewing?**:

**Priority (whenever / two weeks / yesterday)**:
Today - seems a straightforward change, shouldn't take too long to review.

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2662)

### CODE REVIEW

**Comment by schlosna** [comment]
Think you might actually want to also apply some excludes here in atlasdb to prevent future issues, see https://github.com/palantir/atlasdb/commit/b61f37fbf72774b50ce28229bc0ad64defc6eedb

**Comment by tboam** [comment]
Rebuilding

File: atlasdb-config/build.gradle [hunk 1]
```diff
@@ -18,7 +18,9 @@ dependencies {
     compile group: 'com.palantir.remoting3', name: 'keystores'
     compile group: 'com.palantir.safe-logging', name: 'safe-logging'
     compile (group: 'com.palantir.tritium', name: 'tritium-lib') {
+        exclude (group: 'com.palantir.remoting3', module: 'tracing')
```
Comment (hsaraogi) [inline atlasdb-config/build.gradle]: If we dont re-add the excludes, how will it be avaialble to the module. Possibly they are coming in transitively but adding it explicitly feels like a safer option, especially as some classes might be used at runtime only?

Comment (datval) [reply atlasdb-config/build.gradle]: There are multiple transitive dependencies, which bring those modules in. I will add anyways to be explicit. Shouldn't be a problem as long as we don't specify the version.

**Comment by datval** [comment]
Can we merge this please?

**Comment by hsaraogi** [comment]
Taking a final look now.

File: atlasdb-config/build.gradle [hunk 2]
```diff
@@ -18,8 +18,12 @@ dependencies {
...
         exclude (group: 'io.dropwizard.metrics', module: 'metrics-core')
+        exclude (group: 'org.hdrhistogram', module: 'HdrHistogram')
     }
+    // compile group: 'com.palantir.remoting3', name: 'tracing'
```
Comment (hsaraogi) [inline atlasdb-config/build.gradle]: These are commented out?

Comment (datval) [reply atlasdb-config/build.gradle]: Forgot, was testing something.

**Comment by hsaraogi** [comment]
@datval you might need to generate locks again. This looks good, thanks!

**Comment by datval** [comment]
Done

**Pull request merged**